docs/config.rst

```diff
@@ -537,6 +537,29 @@ Features
    .. _Python: https://docs.python.org/tutorial/venv.html
    .. _Conda: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html
 
+.. attribute:: LP_ENABLE_RUBY_VENV
+   :type: bool
+   :value: 1
+
+   Display the currently activated RVM_ or RBENV_ virtual environment.
+
+   See also: :attr:`LP_RUBY_RVM_PROMPT_OPTIONS` and
+   :attr:`LP_COLOR_VIRTUALENV`.
+
+   .. _RVM: https://rvm.io/
+   .. _RBENV: https://github.com/rbenv/rbenv
+
+.. attribute:: LP_RUBY_RVM_PROMPT_OPTIONS
```

Comment:
This section should be alphabetically sorted. It doesn't need to come directly after the ENABLE section since they will be hyperlinked to each other.

Reply:
Sorted